modules/virt-creating-a-service-from-a-virtual-machine.adoc

```diff
@@ -82,8 +82,11 @@ spec:
         - name: cloudinitdisk
           cloudInitNoCloud:
             userData: |
-              #!/bin/bash
-              echo "fedora" | passwd fedora --stdin
+             #cloud-config
```

Comment:
I think the block that is nested under `userData: |` is missing an indentation space (this applies to all the YAMLs).

From the contrib guide:

> Use block style for complex strings, such as embedded JSON or text blocks. You can enable block style by specifying | or |- after a field and indenting the field content by two spaces